_i18n/en.yml

```
@@ -30,6 +31,21 @@ global:
   cookie_inform_and_ask_disagree: Disagree
   tag_page_title: Articles with the tag
   tag_cloud: All tags available on the blog
+  blog: Le blog
+  contact: Contact
+  see_you: Welcome
+  explore_skills: Let's explore new knowledge
+  discover_eleven: Discover ElevenLabs
+  eleven_paris: ElevenLabs - Paris
+  eleven_nantes: ElevenLabs - Nantes
+  eleven_address_paris: 15 Avenue de la Grande armée 75016 Paris
```

je pense qu'il faudrait un caractère de séparation. 
```suggestion
  eleven_address_paris: 15 Avenue de la Grande armée, 75016 Paris
```

---

Modifié